functions/src/main/java/org/apache/geode/examples/functions/Example.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.geode.examples.functions;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.function.BiConsumer;
+import java.util.stream.IntStream;
+
+import org.apache.geode.cache.Region;
+import org.apache.geode.cache.client.ClientCache;
+import org.apache.geode.cache.client.ClientCacheFactory;
+import org.apache.geode.cache.client.ClientRegionShortcut;
+import org.apache.geode.cache.execute.Execution;
+import org.apache.geode.cache.execute.FunctionService;
+import org.apache.geode.cache.execute.ResultCollector;
+
+public class Example implements BiConsumer<Region<Integer, String>, Execution> {
+  int maximum;
+
+  Set<Integer> primes = new HashSet<>();
+
+  public static void main(String[] args) {
+    // connect to the locator using default port 10334
+    ClientCache cache = new ClientCacheFactory().addPoolLocator("127.0.0.1", 10334)
+        .set("log-level", "WARN").create();
+
+    // create a local region that matches the server region
+    Region<Integer, String> region =
+        cache.<Integer, String>createClientRegionFactory(ClientRegionShortcut.CACHING_PROXY)
+            .create("example-region");
+
+    Execution execution = FunctionService.onRegion(region);
+    new Example().accept(region, execution);
+    cache.close();
+  }
+
+  public Example() {
+    this(100);
+  }
+
+  public Example(int maximum) {
+    this.maximum = maximum;
+  }
+
+  public Set<Integer> getPrimes() {
+    return primes;
+  }
+
+  @Override
+  public void accept(Region<Integer, String> region, Execution execution) {
```

Comment:
Maybe this would be better as a method that returned a list of primes? This example is doing a bunch of stuff to maintain a primes field, which seems like it is just for testing. It would be better if this class didn't have any mutable state.

Reply:
I think it is just doing stuff for testing. As I understood it, the intent for these examples was to use `accept()` so that what happens with `../gradlew run` _also_ happens when running the test.